packages/@aws-cdk/aws-eks/lib/cluster.ts

```diff
@@ -78,6 +78,17 @@ export interface ICluster extends IResource, ec2.IConnectable {
    */
   readonly clusterEncryptionConfigKeyArn: string;
 
+  /**
+   * An Open ID Connect Issuer Url
+   * @attribute
+   */
+  readonly clusterOpenIdConnectIssuerUrl: string;
```

Comment:
Not needed anymore. This information should be available via the provider.

Reply:
I'm not sure we want to break ICluster interface backward compatibility. perhaps someone rely on that?

Reply:
`ICluster` doesn't currently have this attribute. Its only present on `Cluster`, which are keeping as is.

Reply:
ah really yeah you're right

I just missed that it is added